src/main/java/com/urbancode/jenkins/plugins/ucdeploy/VersionHelper.java

```diff
@@ -295,6 +295,8 @@ private void setComponentVersionProperties(
                 propSheetDef = compClient.getComponentVersionPropSheetDef(component);
                 propSheetDefId = (String) propSheetDef.get("id");
                 propSheetDefPath = (String) propSheetDef.get("path");
+                propSheetDefPath = propSheetDefPath.replaceAll("/", "&");
```

Comment:
Hi Mitch, 

This is what I sent through the PMR...

I've tried to isolate the issue with the attached test.groovy and uc-uDeployRestClient-1.0-SNAPSHOT.jar files. (This .jar is the same jar that is utilized in the Jenkins Pipeline v2.1.874428 plugin. )  In the test.groovy file, I make four calls to the getPropSheetDefPropDefs() method using both & and / examples. All four tests are working. I cannot reproduce the / error that you are seeing. I suggest that you try this short test on your local instance. It will query your server for the specified component's environment and version properties and print them to the terminal. Note: You will have to update the server, username, password, and component id properties at the top of the groovy file. If it fails, you should see the same 404 Not Found error. The test code is in groovy, however, you can replicate the tests via curl. 

Test Command: groovy -cp "./uc-uDeployRestClient-1.0-SNAPSHOT.jar" test.groovy
Note: If you want to confirm with your own .jar, unzip the Jenkins .hpi file and use that ud-uDeployRestClient-1.0-SNAPSHOT.jar.

If this test fails against your local instance with the same error, then a UCD update will be required. If it succeeds, then the response from the server is an unexpected one. Which we can update UCD or find a better way to parse it within the plugin. However, after talking with the core UCD team, making a hard switch from / to & is likely not the correct resolution.

[Jenkins Test Code.zip](https://github.com/IBM-UrbanCode/ibm-ucdeploy-pipeline-plugin/files/920458/Jenkins.Test.Code.zip)

Hope this helps! Also, what version of UCD are you on? Do you have any iFixes/patches?

-Nick